src/content/docs/opentelemetry/get-started/opentelemetry-get-started-intro.mdx

```diff
@@ -16,15 +16,15 @@ redirects:
   - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start
   - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup
   - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro
-freshnessValidatedDate: never
+freshnessValidatedDate: 2024-09-10
 ---
 
-The OpenTelemetry project has a broad scope including [API and SDK specifications](https://opentelemetry.io/docs/specs/otel/), [instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/), the [Collector](https://opentelemetry.io/docs/collector/), [semantic conventions](https://opentelemetry.io/docs/specs/semconv/), the [OTLP](https://opentelemetry.io/docs/specs/otlp/) protocol, and more. New Relic aims to have first-class support for OpenTelemetry and has a wide variety of platform capabilities that make it easy to use OpenTelemetry tools.
+While OpenTelemetry offers a broad scope encompassing core components like APIs, instrumentation libraries, the Collector, semantic conventions, and the OTLP protocol (see the OpenTelemetry Project for details), New Relic prioritizes seamless integration, making it easy to leverage OpenTelemetry tools.
```

Comment:
I'm missing a link to the OpenTelemetry project here, maybe because I'm not familiar with this feature...